lib/utils.js

```diff
@@ -284,3 +284,19 @@ utils.setNotProvidedParams = (variables, val, key, notProvidedChannelParams, not
         : missingChannelParams);
   }
 };
+
+/**
+ * returns default schema format for a given asyncapi version
+ *
+ * @function getDefaultSchemaFormat
+ * @private
+ * @param  {String} asyncapiVersion AsyncAPI spec version. 
+ */
+utils.getDefaultSchemaFormat = (asyncapiVersion) => {
+  const schemaMatrix = {
+    '2.0.0': 'application/vnd.aai.asyncapi;version=2.0.0',
+    '2.1.0': 'application/vnd.aai.asyncapi;version=2.1.0',
+  };
+
+  return schemaMatrix[String(asyncapiVersion)];
```

Comment:
Why not just this?

```suggestion
  return `application/vnd.aai.asyncapi;version=${asyncapiVersion}`;
```

Reply:
I somehow thought matrix looks cleaner 🤷🏼‍♂️ but also fine to do it this way